### ISSUE TITLE
Why using AUROC vs regular accuracy

### ISSUE THREAD

**Issue by hossam-zaki** [huntforgz/cs541dlfinalproject_chexnet#1]
Hello! I am working on an implementation of this, and I came across your medium article. You guys used Binary Cross Entropy Loss, and I am wondering why you used this as your loss function. Also, what accuracy metric did you use, and why did you use AUROC as your metric?
